docs/release-notes.rst

```diff
@@ -7,7 +7,7 @@ Release notes
 
 Release 0.11.6 (unreleased)
 ===========================
-
+- `PR 767 <https://github.com/uber/petastorm/pull/767>`_: Reader: enable shuffling inside every row group.
```

Comment:
Can you please add just a bit more info? What's the benefit and which arg to use to enable it. Also, please mention deprecated argument.

Reply:
addressed